src/features/Send/useCases/ListAmountsToSend/AddToken/SelectTokenFromListScreen.tsx

```diff
@@ -5,23 +5,28 @@ import {StyleSheet, TouchableOpacity, View} from 'react-native'
 
 import {Boundary, Spacer, Text} from '../../../../../components'
 import {AmountItem} from '../../../../../components/AmountItem/AmountItem'
+import {NftImageGallery} from '../../../../../components/NftImageGallery'
 import {TxHistoryRouteNavigation} from '../../../../../navigation'
 import {useSearch, useSearchOnNavBar} from '../../../../../Search/SearchContext'
 import {useSelectedWallet} from '../../../../../SelectedWallet/Context/SelectedWalletContext'
 import {sortTokenInfos} from '../../../../../utils'
 import {YoroiWallet} from '../../../../../yoroi-wallets/cardano/types'
 import {limitOfSecondaryAmountsPerTx} from '../../../../../yoroi-wallets/contants'
-import {useBalances, useTokenInfos} from '../../../../../yoroi-wallets/hooks'
+import {useBalances, useNfts, useTokenInfos} from '../../../../../yoroi-wallets/hooks'
 import {TokenInfo} from '../../../../../yoroi-wallets/types'
 import {Amounts, Quantities} from '../../../../../yoroi-wallets/utils'
-import {filterAssets} from '../../../common/filterAssets'
+import {filterByFungibility} from '../../../common/filterByFungibility'
+import {filterBySearch} from '../../../common/filterBySearch'
+import {NoAssetFounfImage} from '../../../common/NoAssetFoundImage'
 import {useSelectedSecondaryAmountsCounter, useSend, useTokenQuantities} from '../../../common/SendContext'
 import {useStrings} from '../../../common/strings'
-import {EmptySearchResult} from './Show/EmptySearchResult'
 import {MaxAmountsPerTx} from './Show/MaxAmountsPerTx'
 
+export type Fungibility = 'all' | 'ft' | 'nft'
```

Comment:
I'm leaning towards `FungibilityFilter`, cuz fungibility is boolean, it's or it's not.